module/Company/src/Company/Controller/AdminController.php

```diff
@@ -256,6 +257,70 @@ public function editCategoryAction()
 
         return $vm;
     }
+
+    /**
+     * Action that displays a form for editing a label
+     *
+     *
+     */
+    public function editLabelAction()
+    {
+        // Get useful stuff
```

Comment:
If your comment is like this, you can just as well leave it out. Either the comment should be improved, or the code is trivially self-explanatory.

Reply:
The code, and related comments are mainly adjusted from the Category methods/objects. The code is self-explanatory enough.